src/algebra/module/basic.lean

```diff
@@ -151,6 +151,14 @@ section add_comm_group
 
 variables (R M) [semiring R] [add_comm_group M]
 
+instance add_comm_group.int_semimodule : semimodule ℤ M :=
```

Comment:
I suspect we should have an additional comment here explaining why the "corresponding" `add_comm_group.int_module` doesn't exist, or we'll have people wanting to add it back in.

Reply:
I have changed the instance name to `int_module`, and added a line in the docstring for `semimodule` explaining that `semimodule` and `module` are the same thing.